src/Game/Player/MarioActor.cpp

```diff
@@ -1651,3 +1651,12 @@ TVec3f* MarioActor::getShadowPos() const {
 XanimeResourceTable* MarioActor::getResourceTable() const {
     return mMarioAnim->mResourceTable;
 }
+
+void MarioActor::setPunchHitTimer(unsigned char pPunchHit) {
```

Comment:
Function parameters should only be prefixed with `p` if they are a pointer variable. A better name would be `punchHitTime`.